src/Integration.Vsix/CFamily/FileConfig.cs

```diff
@@ -77,8 +77,9 @@ public static FileConfig TryGet(ILogger logger, ProjectItem dteProjectItem, stri
                     // Not supported
                     return null;
                 }
-
-                var (vcProject, vcConfig, vcFileSettings) = loadedConfig.Value;
```

Comment:
@rita-g-sonarsource I reverted this part and used the ugly old syntax as it failed to compile on the CI. Is there something that we can do about it?

Reply:
Great, Thanks!
`we use PascalCase for method names and properties`
Maybe it is a good idea to enable: https://rules.sonarsource.com/csharp/tag/convention/RSPEC-100 ?

Reply:
Good idea! Done 👍